src/components/display/components/ProgressBarComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet, Text } from 'react-native'
+import React from 'react'
+import { ProgressArgs } from 'display-api'
+import ProgressCircle from 'react-native-progress-circle'
+import colors from '../../../utils/colors'
+
+export const ProgressBarComponent = ({ value, max, inner_label, thickness, color }: ProgressArgs) => {
+	const percentage = (100 * value) / max
+
+	return (
+		<ProgressCircle
+			percent={percentage}
+			radius={40}
+			borderWidth={parseInt(thickness)}
+			color={color}
+			shadowColor='#E8E8E8'
+		>
+			<Text style={styles(color).text}>{inner_label}</Text>
```

Comment:
Can you make the inner label more like this:
![Screen Shot 2021-03-14 at 15 42 50](https://user-images.githubusercontent.com/32454680/111083504-f5b64700-84db-11eb-8f65-c38c9351a1c0.png)
(showing the label as well as the value)